lib/rust/ensogl/core/Cargo.toml

```diff
@@ -33,6 +33,7 @@ bit_field = { version = "0.10.0" }
 console_error_panic_hook = { version = "0.1.6" }
 enum_dispatch = { version = "0.3.6" }
 failure = { version = "0.1.5" }
+hex = { version = "0.4.2" }
```

Comment:
Are we importing the whole new library just for 1 function `from_hex`? Please implement it instead.